libs/components/forms/src/lib/modules/file-attachment/file-attachment.component.ts

```diff
@@ -366,9 +373,20 @@ export class SkyFileAttachmentComponent
   }
 
   public deleteFileAttachment(): void {
+    const fileName = this.value?.file.name;
     this.value = undefined;
-    this.#changeDetector.markForCheck();
     this.#emitFileChangeEvent(this.value);
+
+    // Safety check
+    // istanbul ignore else
```

Comment:
You can combine these comments into one:
```
/* istanbul ignore else: safety check */
```

Reply:
Done